src/runtime-rs/src/shim/src/config.rs

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2021 Alibaba Cloud
+// Copyright (c) 2021 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use std::path::{Path, PathBuf};
+use virtcontainers::config::TomlConfig;
+
+const SANDBOX_CONFIG_PATH_KEY: &str = "io.katacontainers.config_path";
```

Comment:
Isn't this an annotation name? If so, the variable name should reflect that.

Reply:
Fixed in the first revision

Reply:
@dgibson It follows existing Kata logic. See https://github.com/kata-containers/kata-containers/blob/39cf2b27c1cbd48cfd2f0057d04581e8e32ce9c4/src/runtime/pkg/containerd-shim-v2/create.go#L194-L204